web/src/pages/api/public/dataset-run-items.ts

```diff
@@ -93,6 +105,41 @@ export default withMiddlewares({
         },
       });
 
+      /********************
+       * ASYNC RUN ITEM EVAL *
+       ********************/
+
+      if (redis && env.NEXT_PUBLIC_LANGFUSE_CLOUD_REGION) {
+        const queue = DatasetRunItemUpsertQueue.getInstance();
+        if (queue) {
+          await queue.add(
+            QueueJobs.DatasetRunItemUpsert,
+            {
+              payload: {
+                projectId: auth.scope.projectId,
+                type: "dataset" as const,
+                datasetItemId: datasetItemId,
+                traceId: finalTraceId,
+                observationId: observationId ?? undefined,
+              },
+              id: randomUUID(),
+              timestamp: new Date(),
+              name: QueueJobs.DatasetRunItemUpsert as const,
+            },
+            {
+              attempts: 3, // retry 3 times
+              backoff: {
+                type: "exponential",
+                delay: 1000,
+              },
+              delay: 90000, // 90 seconds
```

Comment:
This FK needs to go away for V3 anyways. I think removing this would be an option. Can you check whether this breaks anything in the eval queue downstream (also for the current evals)

Reply:
Will do, sounds good 🙌